wire-ios/sourcery/config.yml

```diff
@@ -0,0 +1,10 @@
+sources:
```

Comment:
is it a part of this PR?

Reply:
Yes! We needed to add sorcery to Wire-iOS so I could generate mocks for the tests.